Polyhedron/demo/Polyhedron/Scene_polyhedron_item.h

```diff
@@ -82,6 +85,9 @@ class SCENE_POLYHEDRON_ITEM_EXPORT Scene_polyhedron_item
     Polyhedron*       polyhedron();
     const Polyhedron* polyhedron() const;
 
+    Polyhedron* facegraph(){return polyhedron();}
```

Comment:
Either you chose Qt naming guidelines: `faceGraph`, or you choose CGAL ones: `face_graph`.